DataCollector/DoctrineDataCollector.php

```diff
@@ -115,8 +117,21 @@ public function collect(Request $request, Response $response, ?Throwable $except
                         continue;
                     }
 
-                    $classErrors                        = $validator->validateClass($class);
-                    $entities[$name][$class->getName()] = $class->getName();
+                    $classErrors = $validator->validateClass($class);
+                    try {
+                        $r                                  = new ReflectionClass($class->getName());
+                        $entities[$name][$class->getName()] = [
+                            'class' => $class->getName(),
+                            'file' => $r->getFileName(),
+                            'line' => $r->getStartLine(),
+                        ];
+                    } catch (ReflectionException $e) {
+                        $entities[$name][$class->getName()] = [
+                            'class' => $class->getName(),
+                            'file' => 'n/a',
```

Comment:
this will go into URL and PhpStorm will probably fail to open it, am I wrong? We could add some checking in template, but I would rather we remove whole try catching and see in real world if we actually ever need it.

Reply:
One of the tests fails since it has a non existing class. Not on pc now so can’t check which one, but how should we handle this?

Reply:
It fails because entity fqcs we configured in DoctrineDataCollectorTest don't exist, let's fix that.

Reply:
So you suggest to just create an empty class for it?

Reply:
It can even be any existing class, like stdClass, doesn't have to be valid entity class

Reply:
Ok let me try this tomorrow. I like the idea

Reply:
->getReflectionClass fixes this actually 😄